app/uk/gov/hmrc/excisemovementcontrolsystemapi/config/AppConfig.scala

```diff
@@ -43,6 +43,10 @@ class AppConfig @Inject() (config: Configuration, servicesConfig: ServicesConfig
     .getOptional[String]("mongodb.movementArchive.TTL")
     .fold(Duration.create(40, DAYS))(Duration.create(_).asInstanceOf[FiniteDuration])
 
+  lazy val miscodedMovementArchiveTTL: Duration = config
```

Comment:
We changed it to 40 because movements were timing out that hadn't actually been received by users, due to the bugs. Once everything has settled we'll change it back to 30.